Makefile

```diff
@@ -78,6 +78,10 @@ install: build
 	install -m 755 target/release/spice ~/.spice/bin/spice
 	install -m 755 target/release/spiced ~/.spice/bin/spiced
 
+.PHONY: install-full
```

Comment:
Better to have task name with `models` in name (`install-with-models`?). otherwise it can confuse as it is not actually a full/all features installation